internal/errors/error.go

```diff
@@ -4,20 +4,20 @@ import "fmt"
 
 // Error is a base error that implement scw.SdkError
 type Error struct {
-	str string
-	err error
+	Str string
+	Err error
```

Comment:
Why make it public ? this is why `Unwrap` is for
Same question for `Str`

Reply:
This is an internal package. It will allow to add error helpers in `scw`.